spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -90,7 +90,8 @@
 
 	/**
 	 * Properties in form {@literal key=value} that should be added
-	 * to the broker config before runs.
+	 * to the broker config before runs.  Properties may contain property placeholders.
+	 * that will be resolved by the Spring {@code Environment}.
```

Comment:
The sentence must start from Capital letter and should be in the present time:

> They are resolved against the Spring `{@code Environment}`
including `{@code @TestPropertySource}`.